mediawikiapi.lua

```diff
@@ -111,7 +112,10 @@ function MediaWikiApi.uploadfile(filepath, pagetext, filename, overwrite)
     text = pagetext,
     comment = 'Uploaded with dtMediaWiki',
     token = MediaWikiApi.getEditToken(),
-    file = {filename="whatevs", file = file_handler:read("*all")},
+    file = {
+        filename = "whatevs",
+        data = file_handler:read("*all"),
```

Comment:
This was the only change required to make it work with vanilla multipart-post (version 1.1; tested w/ Arch Linux)